spec/factories/switch.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :switch
```

Comment:
Same here

Reply:
https://github.com/ManageIQ/manageiq-ui-classic/pull/3933/files#diff-c656f77797c57a2c5dc0688b27ef5565R15

Reply:
Well, I thought we need a *parent* factory if I'm creating a specific one...